west.yml

```diff
@@ -84,7 +86,8 @@ manifest:
       path: modules/hal/libmetal
     - name: hal_quicklogic
       repo-path: hal_quicklogic
-      revision: 7deb3aab1cc430a7650143f9833cfc8ddad76eb5
+      remote: antmicro
```

Comment:
For testing purposes this should be left pointing to the Zephyr HAL repo.  You just need to change the `revision` too be:

`revision: pull/2/head`

Reply:
More details here: https://docs.zephyrproject.org/latest/guides/modules.html?highlight=modules#process

Reply:
Fixed